### PR TITLE
Add top header for NL and VR behavior page behind a feature flag

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Add a temporary component that serves as a "page" behind a feature flag for the new behavior page
Added a data-driven text in the tiles
Moved the calculation functions to a separate file (could also separate them per function)

### CODE REVIEW

File: packages/app/src/domain/behavior/behavior-page-national.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import { ContentHeader } from '~/components/content-header';
+import { Tile } from '~/components/tile';
+import { TileList } from '~/components/tile-list';
+import { TwoKpiSection } from '~/components/two-kpi-section';
+import { Heading, InlineText, Text } from '~/components/typography';
+import { useFormatAndSortBehavior } from '~/domain/behavior/behavior-logic';
+import { useIntl } from '~/intl';
+import { replaceComponentsInText } from '~/utils/replace-components-in-text';
+interface BehaviourPageNationalProps {
+  data: any;
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/domain/behavior/behavior-page-national.tsx:12]: I would give these proper types.

File: packages/app/src/domain/behavior/behavior-page-safety-region.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import { ContentHeader } from '~/components/content-header';
+import { Tile } from '~/components/tile';
+import { TileList } from '~/components/tile-list';
+import { TwoKpiSection } from '~/components/two-kpi-section';
+import { Heading, InlineText, Text } from '~/components/typography';
+import { useIntl } from '~/intl';
+import { replaceComponentsInText } from '~/utils/replace-components-in-text';
+
+interface BehaviorPageSafetyRegionProps {
+  data: any;
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/domain/behavior/behavior-page-safety-region.tsx:12]: `any` => actual types

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: packages/app/src/domain/behavior/behavior-logic.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  behavior: BehaviorValue
+): {
+  sortedCompliance: BehaviorFormatted[];
+  sortedSupport: BehaviorFormatted[];
+} {
+  const compliance = useFormatBehaviorType(behavior, 'compliance');
+  const support = useFormatBehaviorType(behavior, 'support');
+
+  return sortBehavior(compliance, support);
+}
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/behavior/behavior-logic.tsx:85]: This whole file seems to be very specific to the behavior table component. So you might want to rename this to `useBehaviorTableItems` or something and possibly more it to the be nested inside the table.

Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/behavior/behavior-logic.tsx:85]: The top header also uses the sorted data, so not sure about it to make it part of the table. That was the idea to place it above the table.

**Pull request merged**